src/app/css-variables.ts

```diff
@@ -32,6 +32,13 @@ export default function updateCSSVariables() {
       setCSSVariable('--tiles-per-char-column', nextState.charColMobile);
     }
 
+    // Set a class on the body to control the theme
+    document.body.className = currentState.theme;
+    if (currentState.theme !== nextState.theme) {
+      const themeClass = 'theme-' + nextState.theme;
+      document.body.className = themeClass;
+    }
```

Comment:
```suggestion
    // Set a class on the body to control the theme
    if ($featureFlags.themePicker && currentState.theme !== nextState.theme) {
      const themeClass = 'theme-' + nextState.theme;
      document.body.className = themeClass;
    }
```

since settings are synced between DIM channels, the feature flag needs to be here too.

Not sure what the first assignment to`className` is meant to do

the call to `syncThemeColor` below should probably also be moved into this, and the function needs to read `document.body` because that's what the className is applied to

Reply:
The initial className assignment was a result of my misunderstanding how the settings are saved/applied — i thought settings were persisted server-side and applied cross-browser, but i see now they're all client-side. (I was trying to inject the 'currently saved' theme as a class on app load).